shared/DynamicInterop.TestApp/DynamicInterop.TestApp.csproj

```diff
@@ -1,18 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <ProjectGuid>{46208A0E-F95F-4B1E-A396-0F715BA6B87A}</ProjectGuid>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>0.8.2</Version>
```

Comment:
Did you change anything in the DynamicInterop (and also RDotNet)? If no - let's leave the version as is, so it is clear which state of the original code is used.

Reply:
I'm not 100% sure. We had some tests that were already failing when we cloned it and we disabled them if we found them.

The non-test code may have also changed cosmetically.

If you want to keep the original assembly attributes, just close this PR as not needed.